### PR TITLE
Fix docs: support version

### PR DESCRIPTION
**Pull request by Oluwaseun241**
**Description**
[Fix]  support for Python versions in the README

This PR fixes # 396

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,24 @@ app.start(port=8080)
 
 ```
 
+## Python Version Support
+
+Robyn is compatible with the following Python versions:
+
+> Python >= 3.6
```
Comment (sansyrox) [inline README.md:52]: Thanks for the PR @Oluwaseun241 😄 

But Robyn only supports Python version >= 3.7.

Also, would it be possible to add some sort of version shield? Something like this - https://shields.io/

Comment (Oluwaseun241) [reply README.md:52]: I would look into that and update the version support

**Comment by Oluwaseun241** [comment]
@sansyrox i think everything is resolved

File: README.md [hunk 2]
```diff
@@ -45,6 +45,24 @@ app.start(port=8080)
 
 ```
 
+## ![Python Version Support](https://img.shields.io/badge/Support-Version%20%E2%89%A5%203.7-brightgreen)
```
Comment (sansyrox) [inline README.md:48]: Thanks @Oluwaseun241 . Great work. Just one last suggestion. Can you move the badge to the top level with the other badges?

Comment (Oluwaseun241) [reply README.md]: Done

**Review by sansyrox — APPROVED** [review]
LGTM! Great work! ✨

**Pull request merged**